src/Makefile.test.include

```diff
@@ -59,7 +59,6 @@ BITCOIN_TESTS =\
   test/arith_uint256_tests.cpp \
   test/scriptnum10.h \
   test/addrman_tests.cpp \
-  test/alert_tests.cpp \
```

Comment:
I considered leaving the alert tests in order to test that the node longer responds to network alerts &mdash; but since the code to do that is gone, there was nothing to test.